apache-couchdb-mixin/alerts/alerts.libsonnet

```
@@ -0,0 +1,173 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'ApacheCouchDbAlerts',
```

```suggestion
        name: 'ApacheCouchDBAlerts',
```

I saw for mongodb the `db` was lowercased, for oracledb the `db` was capitalized. I think the style guide https://grafana.com/docs/writers-toolkit/style-guide/capitalization-punctuation/#all-caps-capitalization might allude to it being all caps but not entirely sure.

Just wanted to note this since it seemed a tad inconsistent with what other mixins were doing.

---

fixed